src/components/Pagination/Pagination.tsx

```diff
@@ -93,6 +98,7 @@ export const Pagination = ({
   nextLabel = 'Next page',
   pageLabel = page => `Go to page ${page}`,
   totalLabel,
+  tracking,
```

Comment:
I imagined the tracking of the pagination component differently. Instead of adding tracking to the individual components (prev/next buttons, page select, and page list), I'd add it to the main Pagination component.

Something like this: 

```ts
// Pagination.tsx

const Pagination = ({ onChange, tracking = {}, ...otherProps }) => {
  // Can't use our custom useClickTracker here because it doesn't allow us
  // to add the page number as label. So we implement it from scratch:
  const dispatch = useClickTrigger();
  const handleChange = (pageNumber: number) => {
    const {
      label = pageNumber.toString(),
      component = 'pagination',
      customParameters
    } = tracking;

    if (label) {
      dispatch({ label, component, customParameters });
    }

    onChange(pageNumber);
  }

  // Pass down `handleChange` instead of the current `onChange`
  return (
    <IconButton
      label={previousLabel}
      size="kilo"
      variant="tertiary"
      disabled={currentPage <= 1}
      onClick={() => handleChange(currentPage - 1)}
      css={prevButtonStyles}
    >
    {/* ... */}
  );
};
```